eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/test/Azure.Generator.Mgmt.Tests.csproj

```diff
@@ -9,6 +9,7 @@
     <PackageReference Include="NUnit3TestAdapter" />
     <PackageReference Include="Moq" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
+		<PackageReference Include="Microsoft.TypeSpec.Generator.Input" OverrideVersion ="1.0.0-alpha.20250519.4" />
```

Comment:
```suggestion
		<PackageReference Include="Microsoft.TypeSpec.Generator.Input" VersionOverride="1.0.0-alpha.20250519.4" />
```

Reply:
This should be reverted, reason listed above.

Reply:
We could also temporarily exclude mgmt CI from the DPG trigger if needed.

Reply:
Filed https://github.com/Azure/azure-sdk-for-net/issues/50214

Reply:
> Can't we do the same thing we did last time where we pin using a VersionOverride in Tests.Common?

Because last time we ping MTG.Input to a previous version and Azure.Generator didn't break.
Azure.Core broke this time with a previous version of Input.